pkg/plugins/discovery/k8s/controllers/outbound_converter.go

```diff
@@ -54,13 +55,9 @@ func isHeadlessService(svc *kube_core.Service) bool {
 }
 
 func (p *PodConverter) k8sService(serviceTag string) (*kube_core.Service, uint32, error) {
-	host, port, err := mesh_proto.ServiceTagValue(serviceTag).HostAndPort()
+	name, ns, port, err := ParseService(serviceTag)
 	if err != nil {
-		return nil, 0, errors.Errorf("failed to parse `service` tag %q", serviceTag)
-	}
-	name, ns, err := ParseServiceFQDN(host)
-	if err != nil {
-		return nil, 0, errors.Errorf("failed to parse `service` host %q as FQDN", host)
+		return nil, 0, errors.Errorf("failed to parse `service` host %q as FQDN", serviceTag)
```

Comment:
It is not FQDN anymore, at least not in the classic meaning of the abbreviation.

Reply:
I'll fix the msg in other PR. I need to merge it for now